setup.cfg

```diff
@@ -0,0 +1,3 @@
+[pep8]
+exclude=lib
+max-line-length = 100
```

Comment:
Lets change it to 80 [https://www.python.org/dev/peps/pep-0008/#maximum-line-length](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)